examples/app-with-extensions/src/components/Modal/Modal.tsx

```diff
@@ -1,8 +1,10 @@
+import { DataQuery } from '@grafana/data';
 import { testIds } from 'components/testIds';
 import React, { ReactElement } from 'react';
 
 type Props = {
   panelTitle: string | undefined;
+  targets: DataQuery[] | undefined;
```

Comment:
Maybe it would make sense to somehow (in a dummy way) print these out in the modal body?